package.json

```diff
@@ -25,6 +27,8 @@
   },
   "devDependencies": {
     "bl": ">=0.9.0 <0.10.0-0",
+    "coveralls": "^2.11.2",
```

Comment:
@bcoe true,  but I don't know many library authors that would back port fixes to older minor versions of their library.